wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -119,3 +119,50 @@
 		padding: 0 var(--spacer-30);
 	}
 }
+
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Card Horizontal
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-card--style-horizontal {
+	display: flex;
+	flex-flow: row;
+	align-items: center;
+
+	.c-card__title {
```

Comment:
Please add these styles as a conditional selector within the existing element selectors (like CTA on lines 113-121 above). And please add CASE comments as needed.